queries/vue/injections.scm

```diff
@@ -49,6 +49,30 @@
   (#eq? @_ts "ts")
 )
 
+; <script lang="jsx">
+(
+  (script_element
+    (start_tag
+      (attribute
+        (attribute_name) @_lang
+        (quoted_attribute_value (attribute_value) @_js)))
+    (raw_text) @javascript)
+  (#eq? @_lang "lang")
+  (#eq? @_js "jsx")
+)
```

Comment:
This entire section is unnecessary.
Change the javascript capture somewhere above, change the `(#eq? @_js "js")` to `(#any-of? @_js "js" "jsx")`

Reply:
👌